graylog-storage-opensearch2/pom.xml

```diff
@@ -213,6 +213,9 @@
                             <Graylog-Plugin-Properties-Path>${project.groupId}.${project.artifactId}</Graylog-Plugin-Properties-Path>
                         </manifestEntries>
                     </archive>
+                    <archiverConfig>
+                        <duplicateBehavior>add</duplicateBehavior>
+                    </archiverConfig>
```

Comment:
Let's exclude the duplicate file in the `assembly.xml` descriptor file (adding a comment) so we don't have to reconfigure the archiver. I think that's safer than allowing duplicates.

Reply:
I would also prefer this. How would that look, syntax-wise?

Reply:
You can take a look at https://github.com/Graylog2/graylog2-server/pull/18658/commits/e0040b19ae1c321dd062fcb7943aa9f2de23f692 or wait for the merge and add it to the excludes there

Reply:
cherry picked @moesterheld changes + removed duplicate behaviour, added class to exclusion